src/main/java/net/darksky/project/numberone/App.java

```diff
@@ -0,0 +1,46 @@
+package net.darksky.project.numberone;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+import javax.security.auth.callback.LanguageCallback;
+
+import tk.plogitech.darksky.api.jackson.DarkSkyJacksonClient;
+import tk.plogitech.darksky.forecast.ForecastException;
+import tk.plogitech.darksky.forecast.ForecastRequest;
+import tk.plogitech.darksky.forecast.ForecastRequestBuilder;
+import tk.plogitech.darksky.forecast.ForecastRequestBuilder.Language;
+import tk.plogitech.darksky.forecast.model.Forecast;
+
+public class App {
+	public static void main(String[] args) throws IOException, ForecastException {
+		Scanner scanner = new Scanner(System.in);
+		System.out.println("Welcome to my Forecast application.\r\n" + 
+				"Please provide your current latitude and longtitude:");
+	
+		//ReadAFile readAFile = new ReadAFile("welcome.properties");
```

Comment:
this ReadAFile class is for to read a text that is in the beginning and end of the process. But when I create a .jar file, another user should include that text file in classpath so it will be discomfort for the user. Text will be printed using System.out.println() statement instead of file and this class is removed.